config/master.d/50-reactor.conf

```diff
@@ -1,10 +1,7 @@
 reactor:
   - 'salt/presence/change':
-    - /usr/share/salt/kubernetes/reactor/presence.sls
```

Comment:
@inercia  i wonder what the empty state should do. imho is ok to remove it

Reply:
:+1: